.github/workflows/gradle-publish.yml

```diff
@@ -53,12 +50,68 @@ jobs:
           JRELEASER_GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
           JRELEASER_GPG_PASSPHRASE: ${{ secrets.SIGNING_PASSPHRASE }}
 
-      - name: Publish to OSSRH repository
-        run: ./gradlew publish jreleaserFullRelease
+      - name: Publish to Maven Central Portal
+        id: publish
+        run: ./gradlew publish jreleaserFullRelease --info --stacktrace -Djreleaser.verbose=true
         env:
           JRELEASER_MAVENCENTRAL_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           JRELEASER_MAVENCENTRAL_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
           JRELEASER_GPG_PUBLIC_KEY: ${{ secrets.GPG_PUBLIC_KEY }}
           JRELEASER_GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
           JRELEASER_GPG_PASSPHRASE: ${{ secrets.SIGNING_PASSPHRASE }}
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Query Central Portal for validation errors
+        if: failure()
+        shell: bash
+        env:
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          OSSRH_TOKEN: ${{ secrets.OSSRH_TOKEN }}
+        run: |
+          set -euo pipefail
+          
+          if [[ ! -f build/jreleaser/output.properties ]]; then
```

Comment:
Maybe, worth to extract the path to the file into a variable since you reuse it multiple times below.